### ISSUE TITLE
ImportError 

### ISSUE THREAD

**Issue by miscaandrei** [pinax/pinax#105]
Hi, 
I'm very interested in the project and while I was testing it I has a problem when I wanted to make a pinax zero new app

The code error
```
(pinax)amisca@amisca-laptop pinax$ pinax start zero mysite
Traceback (most recent call last):
  File "/home/amisca/Envs/pinax/bin/pinax", line 9, in <module>
    load_entry_point('pinax-cli==0.4', 'console_scripts', 'pinax')()
  File "/home/amisca/Envs/pinax/local/lib/python2.7/site-packages/pkg_resources.py", line 356, in load_entry_point
    return get_distribution(dist).load_entry_point(group, name)
  File "/home/amisca/Envs/pinax/local/lib/python2.7/site-packages/pkg_resources.py", line 2431, in load_entry_point
    return ep.load()
  File "/home/amisca/Envs/pinax/local/lib/python2.7/site-packages/pkg_resources.py", line 2152, in load
    raise ImportError("%r has no %r attribute" % (entry, attr))
ImportError: <module 'pinax' from '/home/amisca/Envs/pinax/local/lib/python2.7/site-packages/pinax/__init__.pyc'> has no 'main' attribute
```

My pip freeze:
```
Django==1.9.1
argparse==1.2.1
click==5.1
django-appconf==1.0.1
django-bootstrap-form==3.2
django-user-accounts==1.3.1
jsonfield==1.0.3
metron==1.3.7
pinax-cli==0.4
pinax-eventlog==1.1.1
pinax-theme-bootstrap==7.2.0
pytz==2015.7
requests==2.7.0
six==1.10.0
wsgiref==0.1.2
```

**Comment by ossanna16** [comment]
Hey @miscaandrei! Thanks for reporting this error. Would you mind joining our Pinax Slack team (http://slack.pinaxproject.com) and asking your question there? Someone should be able to help you in Slack faster than here :)

**Comment by paltman** [comment]
This is fixed with `pinax-cli==0.5` that was just released.
